endpoints/openrtb2/auction.go

```diff
@@ -544,7 +545,9 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request, labels *metric
 	}
 
 	// Populate any "missing" OpenRTB fields with info from other sources, (e.g. HTTP request headers).
-	deps.setFieldsImplicitly(httpRequest, req, account)
+	if errsL := deps.setFieldsImplicitly(httpRequest, req, account); len(errsL) > 0 {
+		errs = append(errs, errsL...)
+	}
```

Comment:
Debug warnings added here will be ignored / filtered out. Before creating the `AuctionRequest`, the error list is filtered to only warning types.

Not sure why that filter exists however, since fatal errors should be dealt with immediately by ending the auction.